src/migrations/20230822192328-internal-goal-duplicate-cleanup.js

```diff
@@ -2389,23 +2823,43 @@ module.exports = {
+          "xZALGoals" AS ( SELECT 'Goals' AS "table", ${auditColumns.join(', ')} FROM "ZALGoals" z WHERE session_sig = '${sessionSig}' AND dml_by = -1),
+          "xZALGoalResources" AS ( SELECT 'GoalResources' AS "table", ${auditColumns.join(', ')} FROM "ZALGoalResources" z WHERE session_sig = '${sessionSig}' AND dml_by = -1),
+          "xZALGoalFieldResponses" AS ( SELECT 'GoalFieldResponses' AS "table", ${auditColumns.join(', ')} FROM "ZALGoalFieldResponses" z WHERE session_sig = '${sessionSig}' AND dml_by = -1),
+          "xZALActivityReportGoals" AS ( SELECT 'ActivityReportGoals' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportGoals" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALActivityReportGoalResources" AS ( SELECT 'ActivityReportGoalResources' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportGoalResources" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALActivityReportGoalFieldResponses" AS ( SELECT 'ActivityReportGoalFieldResponses' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportGoalFieldResponses" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALObjectives" AS ( SELECT 'Objectives' AS "table", ${auditColumns.join(', ')} FROM "ZALObjectives" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALObjectiveFiles" AS ( SELECT 'ObjectiveFiles' AS "table", ${auditColumns.join(', ')} FROM "ZALObjectiveFiles" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALObjectiveResources" AS ( SELECT 'ObjectiveResources' AS "table", ${auditColumns.join(', ')} FROM "ZALObjectiveResources" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALObjectiveTopics" AS ( SELECT 'ObjectiveTopics' AS "table", ${auditColumns.join(', ')} FROM "ZALObjectiveTopics" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALActivityReportObjectives" AS ( SELECT 'ActivityReportObjectives' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportObjectives" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALActivityReportObjectiveFiles" AS ( SELECT 'ActivityReportObjectiveFiles' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportObjectiveFiles" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALActivityReportObjectiveResources" AS ( SELECT 'ActivityReportObjectiveResources' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportObjectiveResources" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
+          "xZALActivityReportObjectiveTopics" AS ( SELECT 'ActivityReportObjectiveTopics' AS "table", ${auditColumns.join(', ')} FROM "ZALActivityReportObjectiveTopics" z WHERE session_sig = '${sessionSig}'  AND dml_by = -1),
```

Comment:
One of the many times when sticking to unquoted column names (i.e. lowercase) makes life easier later.